policies/roadmap.md

```diff
@@ -0,0 +1,70 @@
+# OpenSSL project roadmap
+
+### Last modified 21 March 2022
+
+This document is the OpenSSL project roadmap. It is a living document
+and is expected to change over time. Objectives and dates should be
+considered aspirational.
+
+## Objectives
+
+Some of these objectives can be achieved more easily and quickly
+than others.
+
+#### Regular releases
+
+The OpenSSL project wants to move to having a regular release interval
+of approximately six months.
+
+#### QUIC
+
+OpenSSL is taking a multi-staged approach to the implementation of the
+[QUIC] transport protocol:
```

Comment:
`s/\[QUIC\] transport protocol/QUIC transport protocol (for reference: [QUIC-TRANSPORT], [QUIC-TLS], [QUIC_RECOVERY])/`

Reply:
Never mind (see pickiness discussion in the other review comment)